.github/workflows/e2e.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright © 2021 Cask Data, Inc.
+# Copyright © 2022 Cask Data, Inc.
```

Comment:
nit: no need to update year when you update a file

Reply:
Sure